lib/gtl_formatter.rb

```diff
@@ -41,17 +88,102 @@ def self.format_cols(view, row, controller)
         celltext = send(special_cases[view.extras[:filename]], view, row, col)
       elsif special_cases[view.extras[:filename]].kind_of?(Hash) && special_cases[view.extras[:filename]][view.col_order[col_idx]].present?
         celltext, span = send(special_cases[view.extras[:filename]][view.col_order[col_idx]], row[col])
+      elsif COLUMN_WITH_IMAGE.keys.include?(col)
```

Comment:
`COLUMN_WITH_IMAGE` vs `COLUMN_WITH_ICON` is confusing.  maybe name this constant `COLUMN_WITH_MULTIPLE_ICONS`?

Reply:
maybe the more important question is: why are we returning 3 icons to the caller?  can't we just return the right icon to remove an entire elsif?